.github/workflows/build.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           profile: default
-          toolchain: "1.85"
+          toolchain: "1.86"
```

Comment:
Due to  https://datafusion.apache.org/library-user-guide/upgrading.html#msrv-updated-to-1-86-0